### ISSUE TITLE
Multiprojects with references fail to load

### ISSUE THREAD

**Issue by eloraiby** [ionide/ionide-vscode-fsharp#984]
Assume the following folder structure:
```
ClassLibA/ClassLibA.fsproj
ClassLibB/ClassLibB.fsproj
AppC/AppC.fsproj where AppC references ClassLibA.fsproj and ClassLibB.fsproj
```
In this case, the projects are stuck in loading, sometimes with weird and different error messages. It works when I `touch ClassLibA/ClassLibA.fsproj` and the rest to force a refresh.

On Linux/Mac with dotnet core 2.1

**Comment by neptunao** [comment]
Has exactly the same issue. This problem forced me to switch from VSCode on Linux to VS2017 on Windows.

**Comment by mvsmal** [comment]
Might be related: #924

**Comment by Krzysztof-Cieslak** [comment]
This should be fixed with new project parsing in Ionide 4.0
